### PR TITLE
tests: Get seven more tests running without skipping on device profiles

### PR DESCRIPTION
**Pull request by Whenning42**
Made fixes to two tests and added a high end device profile to get seven more tests running without being skipped.

The tests now no longer being skipped are:
VkPositiveLayerTest.MultiplaneImageTests
VkPositiveLayerTest.MultiplaneGetImageSubresourceLayout
VkLayerTest.CopyImageSinglePlane422Alignment
VkLayerTest.DescriptorIndexingSetLayout
VkLayerTest.MultiplaneImageLayoutBadAspectFlags
VkLayerTest.CopyImageMultiplaneAspectBits
VkLayerTest.InvalidBarriers

### CODE REVIEW

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -23570,23 +23570,24 @@ TEST_F(VkLayerTest, DescriptorIndexingSetLayoutWithoutExtension) {
...
-            m_instance_extension_names.push_back(instance_ext);
+    if (InstanceExtensionSupported(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME)) {
+        m_instance_extension_names.push_back(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
+    } else {
+        printf("             TEST_SKIPPED: Did not find required instance extension %s; skipped.\n",
+               VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
+        return;
+    }
+    ASSERT_NO_FATAL_FAILURE(InitFramework(myDbgFunc, m_errorMonitor));
+    std::array<const char *, 2> required_device_extensions = {VK_KHR_MAINTENANCE3_EXTENSION_NAME,
```
Comment (tobine) [inline tests/layer_validation_tests.cpp]: Any warnings in Android build will cause a fail and this line is hitting an interesting warning. I believe the fix is to put another set of curly braces around the strings:
`std::array<const char *, 2> required_device_extensions = { {VK_KHR_MAINTENANCE3_EXTENSION_NAME, VK_EXT_DESCRIPTOR_INDEXING_EXTENSION_NAME} };`

File: tests/device_profiles/geforce_1080_ti.json [hunk 2]
```diff
@@ -0,0 +1,2190 @@
...
+            96
+        ],
+        "sparseProperties": {
+            "residencyAlignedMipSize": 0,
+            "residencyNonResidentStrict": 1,
+            "residencyStandard2DBlockShape": 1,
+            "residencyStandard2DMultisampleBlockShape": 1,
+            "residencyStandard3DBlockShape": 1
+        },
+        "subgroupProperties": {
```
Comment (mikew-lunarg) [inline tests/device_profiles/geforce_1080_ti.json]: This "subgroupProperties" inside VkPhysicalDeviceProperties causes the file to fail json-schema validation with devsim_1_0_0.  Removing this section will fix.  You'll also want to change the comments section at the top, to indicate the file has been modified from gpuinfo.org.

I've contacted @SaschaWillems about gpuinfo's reportversion 1.5 and 1.6 not validating with schema_1_0_0.  Hopefully we can get that fixed soon, and support that data by migrating to the devsim_1_1_0 schema (when it is published)

Comment (mikew-lunarg) [reply tests/device_profiles/geforce_1080_ti.json]: A side note: I'd welcome any suggestions re adding json-schema validation to our CI processes.  I've not had much success finding a validator that will easily integrate with our automation.

File: tests/layer_validation_tests.cpp [hunk 3]
```diff
@@ -23570,23 +23570,24 @@ TEST_F(VkLayerTest, DescriptorIndexingSetLayoutWithoutExtension) {
...
 
-    std::array<const char *, 2> reqd_instance_extensions = {
-        {VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME, VK_KHR_MAINTENANCE3_EXTENSION_NAME}};
-    for (auto instance_ext : reqd_instance_extensions) {
-        if (InstanceExtensionSupported(instance_ext)) {
-            m_instance_extension_names.push_back(instance_ext);
+    if (InstanceExtensionSupported(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME)) {
+        m_instance_extension_names.push_back(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
+    } else {
+        printf("             TEST_SKIPPED: Did not find required instance extension %s; skipped.\n",
```
Comment (tobine) [inline tests/layer_validation_tests.cpp]: kSkipPrefix

**Pull request merged**